client/homebrew/editor/editor.jsx

```diff
@@ -79,6 +85,19 @@ const Editor = createClass({
 		};
 	},
 
+	handleControlKeys : function(e){
+		if(!(e.ctrlKey || e.metaKey)) return;
+		console.log(e);
+		const X_KEY = 88;
+		if((!e.shiftKey) && (e.keyCode == X_KEY)) this.brewJump();
```

Comment:
`CTRL + x` is "cut". We definitely don't want to overwrite that.

Reply:
As a suggestion:
`CTRL + ALT + RIGHT` => move brew preview to match editor position
`CTRL+ ALT + LEFT` => move editor to match brew preview position

I don't believe these combinations currently have any existing use, please let me know if I'm mistaken